types/scripts/build-types.ts

```diff
@@ -1,14 +1,22 @@
 import assert from "node:assert";
 import childProcess from "node:child_process";
 import events from "node:events";
-import { readFileSync, readdirSync } from "node:fs";
+import { existsSync, readFileSync, readdirSync } from "node:fs";
 import fs from "node:fs/promises";
 import path from "node:path";
 import prettier from "prettier";
 import ts from "typescript";
 import { SourcesMap, createMemoryProgram } from "../src/program.js";
 
-const OUTPUT_PATH = "types/definitions";
+function getFilePath(f: string) {
```

Comment:
Other than the same function being declared in 2 different files, the PR looks good to me 👍 

Would you mind adding a comment to the `getFilePath()` function that didn't get removed at the end?